build_tools/requirements_conda.txt

```diff
@@ -7,3 +7,4 @@ nltk==3.2.1
 pytest==2.9.1       
 flask==0.12         # server
 requests==2.11.1
+matplotlib==1.5.3
```

Comment:
Oh, i see. I was wondering what you mean by "before the tests are executed in Travis and Appveyor".  Thought adding import of `requirements_pip_extra` (again one more mistype, in my previous comment i called it `requirements_pip`) would make it ok.  
So if we're speaking about changing .yaml file, i'd still run `pip install matplotlib` locally while uncommenting that test code (that will eventually go to examples, right).   

P.S. before pushing the second version of this PR, all these changes i should put on top of new version of `master` branch with a `rebase`, right?